apps/widget-e2e/src/util/Table.ts

```diff
@@ -29,3 +29,23 @@ export function getTableRowCount(srPrefix?: string): number {
     `${selectorPrefix} [data-automationid=DetailsList] [data-automationid=ListCell]`
   ).length;
 }
+
+export function assertRowSelected(
+  columnName: string,
+  columnValue: string,
+  srPrefix?: string
+): void {
+  const selectorPrefix = srPrefix ? `${srPrefix}` : "";
+  const columnsHeaderColumn = `${selectorPrefix} [data-automationid='ColumnsHeaderColumn']`;
+  cy.get(columnsHeaderColumn)
+    .contains(columnName)
+    .parents(columnsHeaderColumn)
+    .invoke("attr", "data-item-key")
+    .then((columnIndex) => {
```

Comment:
This way may be an issue.
The async statement in side "then" may not triggered before test finish.

Probably better to convert to async + await

Reply:
The statements inside will be async, but Cypress will manage async and await by default.